corehq/apps/users/views/__init__.py

```diff
@@ -382,15 +409,66 @@ def web_users(self):
                     web_users.append(user)
         for user in web_users:
             user.current_domain = self.domain
-        web_users.sort(key=lambda x: (x.role_label(), x.email))
-        return web_users
+
+    @allow_remote_invocation
+    def get_users(self, in_data):
+        if not isinstance(in_data, dict):
+            return {
+                'success': False,
+                'error': _("Please provide pagination info."),
+            }
+        try:
+            limit = in_data.get('limit', 10)
+            page = in_data.get('page', 1)
+            skip = limit * (page - 1)
+            query = in_data.get('query')
+
+            web_users_query = self.query_es(limit, skip, query=query)
+            total = web_users_query.get('hits', {}).get('total', 0)
+            results = web_users_query.get('hits', {}).get('hits', [])
+
+            web_users = [WebUser.wrap(w['_source']) for w in results]
+            self.apply_teams_to_users(web_users)  # for roles
+
+            def _fmt_result(domain, u):
+                return {
+                    'email': u.email,
+                    'domain': domain,
+                    'name': u.full_name,
+                    'role': u.role_label(),
+                    'phoneNumbers': u.phone_numbers,
+                    'id': u.get_id,
+                    'editUrl': reverse('user_account', args=[domain, u.get_id]),
+                    'removeUrl': (
+                        reverse('remove_web_user', args=[domain, u.user_id])
+                        if self.request.user.username != u.username else None
+                    ),
+                }
+            web_users_fmt = [_fmt_result(self.domain, u) for u in web_users]
+
+            return {
+                'response': {
+                    'users': web_users_fmt,
+                    'total': total,
+                    'page': page,
+                    'query': query,
+                },
+                'success': True,
+            }
+        except Exception as e:
+            return {
+                'error': e.message,
+                'success': False,
+            }
 
     @property
     @memoized
     def user_roles(self):
         user_roles = [AdminUserRole(domain=self.domain)]
-        user_roles.extend(sorted(UserRole.by_domain(self.domain),
-                                 key=lambda role: role.name if role.name else u'\uFFFF'))
+        user_roles.extend(sorted(
+            UserRole.by_domain(self.domain),
+            key=lambda role: role.name if role.name else u'\uFFFF'
```

Comment:
what does the escape sequence do here?  Is it just `<` everything else or something?